packages/kserve/src/deploy.ts

```diff
@@ -100,16 +100,10 @@ const assembleInferenceService = (
   const annotations = { ...inferenceService.metadata.annotations };
   annotations['openshift.io/display-name'] = name.trim();
   annotations['opendatahub.io/model-type'] = modelType;
```

Comment:
i have no idea how to test this.  i just removed the annotations.  @Gkrumbach07 ?